### ISSUE TITLE
Starting and then stopping an AVCaptureSession anytime before a meeting will result in a popping sound when starting a Chime session

### ISSUE THREAD

**Issue by jatindervrify** [aws/amazon-chime-sdk-ios#430]
**Describe the bug**
- Use an `AVFoundation` `AVCaptureSession` to start capturing audio and then stop the capture session.  Afterwards start a Chime session normally with `try currentMeetingSession.audioVideo.start(audioVideoConfiguration: audioVideoConfig)` and observe that a distinct popping/breaking sound is heard. 
- This is very apparent across many iPhone and iPad test devices so device type does not seem to matter. 
- This may be specific to iOS 15 or greater as the issue doesn't appear present on our iOS 14 test devices.  
- This is reproducible in the demo app using the steps below. 

**To Reproduce**

1. To the demo project add the following file: 

```
import Foundation
import AVFoundation

final class MicrophonePreviewManager: NSObject {
    private let captureSession: AVCaptureSession
    private let sessionQueue: DispatchQueue

    init(captureSession: AVCaptureSession = AVCaptureSession(),
         sessionQueue: DispatchQueue = .global(qos: .utility)) {
        self.captureSession = captureSession
        self.sessionQueue = sessionQueue
    }

    func startCapturing() {
        guard let captureDevice = AVCaptureDevice.default(for: .audio) else { return }

        do {
            let input = try AVCaptureDeviceInput(device: captureDevice)
            let output = AVCaptureAudioDataOutput()

            captureSession.beginConfiguration()
            if captureSession.canAddInput(input) {
                captureSession.addInput(input)
            }

            if captureSession.canAddOutput(output) {
                captureSession.addOutput(output)
            }

            captureSession.commitConfiguration()

            DispatchQueue.main.async {
                self.captureSession.startRunning()
            }

        } catch { }
    }

    func stopCapturing() {
        if captureSession.isRunning {
            self.captureSession.beginConfiguration()
            self.captureSession.inputs.forEach { self.captureSession.removeInput($0) }
            self.captureSession.outputs.forEach { self.captureSession.removeOutput($0) }
            self.captureSession.commitConfiguration()
            self.captureSession.stopRunning()
        }
    }
}

```

2. In `JoiningViewController.swift` add the follow property and override `viewDidAppear`
```
    private let micManager = MicrophonePreviewManager()

    override func viewDidAppear(_ animated: Bool) {
        micManager.startCapturing()
        micManager.stopCapturing()
    }
```
3. Start the demo as you normally would
4. Observer a loud popping sound occurs. You can hear it at the 8 second mark below. It's much louder on actual device. 

https://user-images.githubusercontent.com/41025959/159590355-0b46515d-0790-4384-a809-a04654992654.MP4

**Expected behavior**
When starting a chime session there should be no popping sound

**Logs**
N/A

**Screenshots**
Video attached above

**Test environment Info (please complete the following information):**
 - Device: iPhone 12 Pro, iPad Pro 5th Gen, iPhone 8, M1 iPad Pro
 - OS: iOS 15.3.1
 - Version AmazonChimeSDK: 0.19.2
 - Version AmazonChimeSDKMedia: 0.19.2
 - Can you reproduce this in the demo app? Yes
If you are reporting a crash:
 - Are you using bitcode supported binaries?
 - Please provide full crash logs in the form of a ".crash" or ".ips" file with a binary images section.

**Additional context**
N/A

**Comment by ziyiz-amzn** [comment]
Thank you @jatindervrify for reporting this. 

Have you ever tried to start a Chime meeting session after the stop running completes? e.g. [AVCaptureSessionDidStopRunning](https://developer.apple.com/documentation/foundation/nsnotification/name/1389889-avcapturesessiondidstoprunning) to avoid any potential race condition?

Also, I see some post like [this](https://www.reddit.com/r/iOSBeta/comments/p4qk90/ios15_db5_does_anyone_else_have_crackling_audio/), you may file a support ticket to Apple as well.

**Comment by jatindervrify** [comment]
Thanks for the response @ziyiz-amzn. Yes, I've tried stopping the `AVSessionCapture` well before starting the Chime meeting session. In the code above the `stopRunning` is called in the `viewDidAppear` of the demo's `JoiningViewController.swift`. It's around 20 seconds after stopping the capture in the video above that the Chime session is started the audible popping occurs. 

I looked into this being an iOS issue but when I tried to reproduce it in a small project that captured audio/video while trying a number of different timings with starting/stopping captures I could not reproduce the popping sound.
